### ISSUE TITLE
use_g_set_str: Missleading row number

### ISSUE THREAD

**Issue by A6GibKm** [bilelmoussaoui/gobject-lint#3]
The row at:
```
xdg-desktop-portal/document-portal/document-portal-fuse.c:1975:7
  g_clear_pointer (&path, g_free);
  error: Use g_set_str(&path, resolved_path) instead of g_free and g_strdup
  rule: use_g_set_str
```

suggests this is a false positive, it should instead point towards the next line (1976):
```
      path = g_strdup (resolved_path);
```

ideally, it should show both though.

**Comment by bilelmoussaoui** [comment]
> ideally, it should show both though.

Right

**Comment by bilelmoussaoui** [comment]
Can you try out main now? it should be a lot better
